packages/services/mock-data-service/docker/jvm/Dockerfile.jvm

```diff
@@ -1,11 +1,7 @@
 ####
 # This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
 #
-# Before building the container image run:
-#
-# ./gradlew build
-#
-# Then, build the image with:
+# Build the image with:
 #
 # docker build -f src/main/docker/Dockerfile.jvm -t quarkus/server-scaffold-jvm .
 #
```

Comment:
@dgauldie All this below, is any of that useful? Guessing this is just from a quarkus template just wondering if we can clean it up

Reply:
I was wondering the same thing about all those comments.
Also @Nithos I don't think we need to bother with mock-data-service anymore, so I'm not sure if it's worth keeping all of those scripts.